nntrainer/tensor/tensor_pool.cpp

```diff
@@ -272,6 +282,15 @@ Tensor *TensorPool::create(const std::string &name, const TensorDim &dim,
   return requestTensor(dim, exec_order, lifespan, name, init);
 }
 
+Tensor *TensorPool::view(const std::string &name, const std::string &reference,
+                         const TensorDim &dim,
+                         const std::vector<unsigned int> &exec_order,
+                         TensorLifespan lifespan, const unsigned int offset) {
+  /// @todo rename requestPrerequestedTensor -> view
+  return requestPrerequestedTensor(dim, exec_order, lifespan, name, reference,
```

Comment:
I recommend you to either rename from the start itself or add in the comments that `view` is an alias for `requestPrerequestedTensor`, ensuring that they follow same semantics.

Reply:
Forwarding discussion to https://github.com/nnstreamer/nntrainer/pull/1693#discussion_r744373869